### PR TITLE
test(netpol): check netpol generated output

### PR DESCRIPTION
**Pull request by janisz**
## Description

Add check for generated output of netpol generate so we fail in unit tests not only in bats.

Refs:
- https://github.com/stackrox/stackrox/pull/7139

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

**Comment by janisz** [comment]
@shireenf-ibm  PTAL

File: roxctl/connectivity-map/netpol_test.go [hunk 1]
```diff
@@ -163,6 +169,33 @@ func (d *analyzeNetpolTestSuite) TestAnalyzeNetpol() {
 			expectedAnalysisError: nil,
 			outputToFile:          true,
 			outputFormat:          "json",
+			expectedOutput: `[
```
Comment (dhaus67) [inline roxctl/connectivity-map/netpol_test.go:172]: nit(applies to the other expected output as well): could make this a file within `testdata/...`, but it's not strictly required (I'm not sure how many other test cases and expected outputs we're going to have, if we foresee that it's going to be a few more then we probably should move to `testdata` IMO.

**Review by dhaus67 — APPROVED** [review]

File: roxctl/connectivity-map/netpol_test.go (hunk 1 above)
Comment (janisz) [reply roxctl/connectivity-map/netpol_test.go]: Moved

**Review by vikin91 — APPROVED** [review]
Thank you for improving this!

**Pull request merged**